### DISCUSSION TITLE
Not being able to get elements properties completion (React)

### DISCUSSION THREAD

**Discussion by ndavd** [typescript-language-server/typescript-language-server#811] [Q&A] [answered]
Completion for these used to work fine, now I'm just not getting any completion when I try to get any property:
```tsx
const MyComponent = () => {
  return (
    <div styl... />
  )
}
```
However completion works fine once I'm "inside" a property:
```tsx
const MyComponent = () => {
  return (
    <div style={{ ... }} />
  )
}

```

**Comment by rchl** [comment]
Please provide a reproduction repo that worked before and doesn't work anymore.

Also LSP logs from your editor.

**Reply by ndavd** [reply]
> Please provide a reproduction repo that worked before and doesn't work anymore.

I have been having this issue for a while now, it just so happens that only now I thought about reporting it — No idea when was the last time that these worked.

> Also LSP logs from your editor.

Of course, providing a minimal one where I do the following:
1. Try to autocomplete the `style` property
2. Go inside and actually get some suggestions

[lsp.log](https://github.com/typescript-language-server/typescript-language-server/files/13376950/lsp.log)

**Reply by rchl** [reply]
Can you also provide logs just after starting the starting? I want to see the logs when files are being opened (didOpen is triggered).

**Reply by ndavd** [reply]
Of course, did the same exact thing but now since the editor starts until I close it — [lsp.log](https://github.com/typescript-language-server/typescript-language-server/files/13377210/lsp.log)

**Reply by rchl** [reply, marked as answer]
Didn't recognize it at first but this is an old "issue" - https://github.com/typescript-language-server/typescript-language-server/issues/679#issuecomment-1424701475

Basically your client doesn't advertise support for snippet completions and those are snippet completions in typescript.

Your client most likely supports those and you just need to enable them.

**Reply by ndavd** [reply]
Thanks! Adding the `capabilities` to the language server did it for me!
